fix(performance) : Fix abend on accessing Performance tab on Non Apache server

Udinra opened the pull request:
If you try to access Performance page from Admin it will abend on Nginx powered Elgg installation as get_apache_version function called in the file will not be defined

Udinra committed:
Update generic.php - Error on Non Apache installs

If you try to access Performance page from Admin it will abend on Nginx powered Elgg installation as get_apache_version function called in the file will not be defined

jeabakker commented on views/default/admin/performance/generic.php:
```diff
@@ -26,21 +26,23 @@
-	$value = elgg_echo('status:unavailable');
-	$subtext = '';
+// Check if the Server is Apache before calling apache_get_version else it may error out in server like Nginx
+if (strpos($_SERVER['SERVER_SOFTWARE'],'Apache') !== false) {	
```
why not check on the existence of the function? Your way is (currently) case sensitive, and maybe some other webserver says it's Apache when it isn't (can happen with opensource software)

Udinra replied:
Yeah good suggestion. I will update the code and see Elgg guidelines. That was quick dirty fix to make it working on my personal site

jeabakker replied:
> That was quick dirty fix to make it working on my personal site

understand completely ;) now try to do it the 'right' way ;)

jeabakker reviewed (changes requested):
Thanks for the fix, maybe look at my comment.

Also look at the error's reported by Travis (coding style and commit message format) 

http://learn.elgg.org/en/stable/contribute/code.html#commit-message-format

Udinra committed:
Update generic.php

fix(performance): Fix abend on accessing Performance tab on Non Apache server #12794

jeabakker commented:
please have a look at the lint errors https://travis-ci.org/Elgg/Elgg/jobs/576382512

jeabakker commented:
replaced by #12803
